edx_repo_tools/codemods/django3/setup_file_modernizer.py

```diff
@@ -8,6 +8,7 @@
 
 import click
 
+TROVE_CLASSIFIERS_INDENT_COUNT = 8
```

Comment:
It wouldn be better to declare it as a class variable instead if it is only going to be used inside the `SetupFileModernizer` class.